repo/packages/Z/zeppelin/0/marathon.json.mustache

```diff
@@ -1,5 +1,6 @@
 {
     "id": "/zeppelin",
+    "acceptedResourceRoles": [ "slave_public" ],
```

Comment:
I was thinking that this would be `{{zeppelin.role}}`. The default would be `*` and the user could change it to deploy somewhere else.